src/Hamiltonians/GutzwillerSampling.jl

```diff
@@ -21,16 +21,28 @@ After construction, we can access the underlying Hamiltonian with `G.hamiltonian
 
 ```jldoctest
 julia> H = HubbardMom1D(BoseFS{3}((1,1,1)); u=6.0, t=1.0)
```

Comment:
Even though it worked in the old version, this should probably not have been in the docstring like this
```suggestion
julia> H = HubbardMom1D(BoseFS{3,3}((1,1,1)); u=6.0, t=1.0)
```